deployments/helm/dra-example-driver/values.yaml

```diff
@@ -87,3 +87,14 @@ webhook:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
+
+resourcequota:
+  enabled: false
```

Comment:
I'd vote to enable this for e2e tests for that extra bit of coverage like we do for the webhook which is disabled by default: https://github.com/kubernetes-sigs/dra-example-driver/blob/e82f2915b7af8ade4124f255ec9d847c343ef565/test/e2e/setup-e2e.sh#L36